Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
+      formatador (>= 0.2.4)
+      listen (>= 2.7, < 4.0)
+      lumberjack (>= 1.0.12, < 2.0)
+      nenv (~> 0.1)
+      notiffany (~> 0.0)
+      pry (>= 0.13.0)
+      shellany (~> 0.0)
+      thor (>= 0.18.1)
+    guard-compat (1.2.1)
+    guard-rspec (4.7.3)
+      guard (~> 2.1)
+      guard-compat (~> 1.1)
+      rspec (>= 2.99.0, < 4.0)
+    listen (3.7.1)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
+    lumberjack (1.2.8)
+    method_source (1.0.0)
+    mock_redis (0.31.0)
+      ruby2_keywords
+    nenv (0.3.0)
+    notiffany (0.1.3)
+      nenv (~> 0.1)
+      shellany (~> 0.0)
+    pry (0.14.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    rake (13.0.6)
+    rb-fsevent (0.11.1)
+    rb-inotify (0.10.1)
+      ffi (~> 1.0)
+    redis (4.6.0)
```

Comment:
### [redis](https://github.com/redis/redis-rb)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 3.2.0 → 4.6.0

[_[change-log](https://github.com/redis/redis-rb/blob/master/CHANGELOG.md), [source-code](https://github.com/redis/redis-rb/tree/v4.6.0), [gem-diff](https://my.diffend.io/gems/redis/3.2.0/4.6.0)_]





<details>
<summary>Commits</summary>

A change of **673** commits. See the full changes on [the compare page](https://github.com/redis/redis-rb/compare/v3.2.0...v4.6.0).

These are the first 10 commits:
- (0559b50) [Fix Sentinel authentication and resolve #855](https://github.com/redis/redis-rb/commit/0559b503f9303e9716957bdbea2f731006d549b2)
- (d21527d) [Merge pull request #856 from supercaracal/fix-sentinel-auth](https://github.com/redis/redis-rb/commit/d21527d218079aeb91b7d6488a39c17295bfba1d)
- (cc6eadf) [Fix a typo](https://github.com/redis/redis-rb/commit/cc6eadfc83860f5b25c7190faa10ad7bca3dd297)
- (a060e8c) [Update rubygems version](https://github.com/redis/redis-rb/commit/a060e8c0f5586cafa5c09e8bfbd99482c62b0690)
- (f4fd4f7) [Merge pull request #857 from supercaracal/fix-a-typo](https://github.com/redis/redis-rb/commit/f4fd4f7c005283804424be3f8f4fa8f58bd11093)
- (331fadc) [Merge pull request #858 from supercaracal/update-rubygems-ve…](https://github.com/redis/redis-rb/commit/331fadc579684a772fccb31afd6ff8e6bc1717f9)
- (d6a2fd5) [Use nonblocking IO for SSL socket connect.](https://github.com/redis/redis-rb/commit/d6a2fd57b1fdcadd37d8e4f13df46532e2b278f2)
- (9101933) [Release 4.1.2](https://github.com/redis/redis-rb/commit/9101933c84d11f08794466ad0903467958e014ba)
- (5c6ca51) [Reduce timeout for ssl test](https://github.com/redis/redis-rb/commit/5c6ca51ee9f87fd9705e78f5cf8a5300d366d2db)
- (38f2b07) [Add "See Also" section to README.](https://github.com/redis/redis-rb/commit/38f2b072e4aca7c0eddd5d3ba72d6d9280ac2dfb)


</details>